modules/social_features/social_group/social_group.module

```diff
@@ -2091,6 +2091,17 @@ function _social_group_delete_group() {
   }
 }
 
+/**
+ * Get all public groups.
+ */
+function social_group_get_all_public_groups(): array {
```

Comment:
Could this be moved to a service, whether in social_core or social_group itself?

Reply:
@navneet0693 Since all group types will be removed in the future I think we can leave it here so far. At least we have the same functions for all and open groups `social_group_get_all_open_groups` and `social_group_get_all_groups`.

Reply:
Check okay.